app/processing/settlement/get-settled-settlement-by-settlement-id.js

```diff
@@ -8,7 +8,8 @@ const getSettledSettlementBySettlementId = async (settlementId, transaction) =>
       'invoiceNumber',
       'reference',
       'settled',
-      'settlementDate'
+      'settlementDate',
+      'value'
```

Comment:
Need the value so we can offset against previous to calculate current payment value.